velox/type/tests/DecimalTest.cpp

```diff
@@ -490,5 +506,44 @@ TEST(DecimalTest, rescaleReal) {
   assertRescaleRealFail(
       INFINITY, DECIMAL(10, 2), "The input value should be finite.");
 }
+
+TEST(DecimalTest, convertToStringView) {
+  testConvertToStringView<int64_t>(12, 10, 0, 11, "12");
+  testConvertToStringView<int64_t>(12, 10, 1, 12, "1.2");
+  testConvertToStringView<int64_t>(12, 10, 3, 12, "0.012");
+  testConvertToStringView<int64_t>(-12, 10, 3, 12, "-0.012");
+  testConvertToStringView<int64_t>(12, 5, 5, 8, "0.00012");
+  testConvertToStringView<int64_t>(-12, 5, 5, 8, "-0.00012");
+  testConvertToStringView<int64_t>(-12, 5, 5, 8, "-0.00012");
+  testConvertToStringView<int64_t>(
+      DecimalUtil::kShortDecimalMax, 18, 0, 19, std::string(18, '9'));
+  testConvertToStringView<int64_t>(
+      DecimalUtil::kShortDecimalMin, 18, 0, 19, "-" + std::string(18, '9'));
+
+  testConvertToStringView<int128_t>(
+      HugeInt::build(0xFFFFFFFFFFFFFFFFull, 0),
```

Comment:
Perhaps use HugeInt::parse to improve the readability.

Reply:
Some meta employees suggest me to use HugeInt::build to accelerate.
https://github.com/facebookincubator/velox/pull/4613#discussion_r1242690466

Reply:
That comment doesn't seem right. In tests, we should favor readability as top performance is not important.

FYI, @karteekmurthys used to work for IBM (Ahana), but I believe he recently joined Meta.

Reply:
Yes, I also agree readability has high priority.

For this case, do we need to use `HugeInt::parse`? It is nearly complex to `castToString`

Reply:
Can we add a function to DecimalUtil or HugeInt to support readable and simple implement?
```
  static int128_t combine(int64_t hi, int64_t lo) {
    return hi * 1000,000,000,000,000,000 + lo;
  }
```

Reply:
@jinchengchenghh Sounds reasonable. Curious, what would the usage of this look like.